examples/riot/pkg_libcoap/Makefile.include

```diff
@@ -0,0 +1,6 @@
+INCLUDES += -I$(PKGDIRBASE)/libcoap \
+            -I$(PKGDIRBASE)/libcoap/include
+
+# definitions for libcoap-contrib
```

Comment:
I suggest removing these comments. I do not see future support for what I did with that.